manifests/init.pp

```diff
@@ -484,6 +484,11 @@
     fail('You need to specify a value for javahome')
   }
 
+  $change_dbpassword = versioncmp($version, '10.3.0') ? {
```

Comment:
Puppet does not allow reassignments.

There are two possible solutions:

1: do this in the parameter list, default to false.
2: use a variable like $real_change_dbpassword, to assign this to and use this one afterwards